aidants_connect/settings.py

```diff
@@ -234,3 +236,15 @@
         "/loisirs.png",
     },
 }
+
+# Magic Auth
+MAGICAUTH_EMAIL_FIELD = "email"
+MAGICAUTH_FROM_EMAIL = "test@domain.user"
+MAGICAUTH_LOGGED_IN_REDIRECT_URL_NAME = "dashboard"
+MAGICAUTH_LOGIN_VIEW_TEMPLATE = "login/login.html"
+MAGICAUTH_EMAIL_SENT_VIEW_TEMPLATE = "login/email_sent.html"
+
+EMAIL_BACKEND = "django.core.mail.backends.filebased.EmailBackend"
```

Comment:
Est-ce que ce réglage est utile seulement en test?

Reply:
I see what you mean de crayon ✏️